src/test/ui/rfc-2632-const-trait-impl/stability.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(incomplete_features)]
+#![feature(allow_internal_unstable)]
+#![feature(const_add)]
+#![feature(const_trait_impl)]
+#![feature(staged_api)]
+
+pub struct Int(i32);
+
+#[rustc_const_unstable(feature = "const_add", issue = "none")]
+impl const std::ops::Add for Int {
+    type Output = Self;
+
+    fn add(self, rhs: Self) -> Self {
```

Comment:
What happens if someone annotates this function with `rustc_const_stable`? I think that should not be permitted at all.